samples/v1.0/Scenarios/FoodOrder.json

```diff
@@ -21,17 +21,17 @@
 			"images": [
 				{
 					"type": "Image",
-					"url": "https://contososcubademo.azurewebsites.net/assets/steak.jpg",
```

Comment:
is this change necessary for this PR?

Reply:
this isn't needed to fix the 6294, but i wanted put sample updates together.